metadata.json

```diff
@@ -40,6 +40,12 @@
       ]
     }
   ],
+  "requirements": [
+    {
+      "name": "puppet",
+      "version_requirement": ">= 3.8.0 < 4.0.0"
```

Comment:
Does this version requirement apply to the master that I would install this on to later upgrade my agents?